hummingbird/ml/operator_converters/xgb.py

```diff
@@ -135,7 +135,15 @@ def convert_sklearn_xgb_regressor(operator, device, extra_config):
         feature_names = operator.raw_operator.get_booster().feature_names
         if feature_names is not None:
             extra_config[constants.FEATURE_NAMES] = feature_names
-    base_prediction = operator.raw_operator.base_score
+    import json
+
+    def get_basescore(model):
+
+        """Get base score from an XGBoost sklearn estimator."""
+        base_score = float(json.loads(model.get_booster().save_config())["learner"]["learner_model_param"]["base_score"])
```

Comment:
Can you do the same without having to dump the model into json and then load it again? Like can you access the `base_score` from `operator.raw_operator.lerner.lernel_model_parm`?